editor/code_editor.cpp

```diff
@@ -944,6 +944,8 @@ void CodeTextEditor::_complete_request() {
 		Color font_color = completion_font_color;
 		if (e.insert_text.begins_with("\"") || e.insert_text.begins_with("\'")) {
 			font_color = completion_string_color;
+		} else if (e.insert_text.begins_with("##") || e.insert_text.begins_with("///")) {
+			font_color = completion_doc_comment_color;
```

Comment:
Hmm, when would autocompletion ever use `##` or `///` as a prefix? And why not `/**`? I see this already exist for normal comment delimiters, but I don't know where those might appear in autocomplete either...

Reply:
I don't remember comments ever being auto-completed either. But I didn't dare remove this.

Reply:
* With #78941 and #79761 this might make sense.